torchinductor/scheduler.py

```diff
@@ -118,15 +124,24 @@ def get_priority(self):
 
 
 class ExternKernelSchedulerNode(BaseSchedulerNode):
+    def __init__(self, scheduler: "Scheduler", node: ir.ExternKernel, group_fn):
+        super().__init__(scheduler, node)
+        if type(node) in TemplateKernels and node.get_device == "cuda":
```

Comment:
You can use https://github.com/pytorch/torchdynamo/blob/e5e2366821c0e26a2791508d0359598e1d6e030e/torchinductor/ir.py#L112